src/converters/utils/calculations/getApproveTime.ts

```
@@ -42,12 +44,13 @@
   );
 
   const isApproved =
-    statuses.some((status) => status.state === "approved") &&
+    statuses.filter((status) => status.state === "approved").length >=
+      parseInt(getValueAsIs("REQUIRED_APPROVALS")) &&
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)